aws/aws-how-to/kubernetes/deploy-ubuntu-pro-cluster-with-eks-pro-ami.rst

```diff
@@ -24,18 +24,39 @@ Create a ``config.yaml`` with the following content:
 
 ..  code-block:: yaml
 
+    ---
     apiVersion: eksctl.io/v1alpha5
     kind: ClusterConfig
+    
     metadata:
-        name: your-pro-cluster
-        region: us-east-1
-    NodeGroups:
-    - name: ng-procluster
-      instanceType: m5.large
-      amiFamily: UbuntuPro2204
-      desiredCapacity: 2
-
-This config file will allow you to use ``eksctl`` to create an EKS cluster and node groups. By specifying ``amiFamily: UbuntuPro2204``, we ensure that the EKS Pro AMI will be used during creation and deployment.
+      name: my-pro-cluster
+      region: us-east-1
+      version: '1.31'
+    
+    iam:
+      withOIDC: true
+    
+    nodeGroups:
+      - name: ng-ubuntu-pro-2404
+        instanceType: m5.large
+        desiredCapacity: 3
+        amiFamily: UbuntuPro2404
+        iam:
+           attachPolicyARNs:
+              - arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy
+              - arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly
+              - arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
+              - arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy
+        ssh:
+            allow: true
+            publicKeyName: myKeyPair
+
+
+This config file will allow you to use ``eksctl`` to create an EKS cluster and node groups. By specifying ``amiFamily: UbuntuPro2404``, we ensure that the EKS Pro AMI will be used during creation and deployment.
+
+To use an Ubuntu specific Ubuntu Pro AMI version, set ``amiFamily`` to one of these choices:
+- ``UbuntuPro2204`` for EKS version >= 1.29 and <=1.32
```

Comment:
* we don't know yet when we end to add new EKS versions to 22.04 . can we just drop `and <=1.32` ?
* 1.32 is not yet released. so this might be confusing